runtime/solo/Cargo.toml

```diff
@@ -64,12 +64,13 @@ pallet-sidechain = { default-features = false, git = "https://github.com/integri
 pallet-teerex    = { default-features = false, git = "https://github.com/integritee-network/pallets", rev = "4d1b0e9" }
 
 # Ajuna pallets
-ajuna-common              = { default-features = false, path = "../../ajuna-common" }
-ajuna-primitives          = { default-features = false, path = "../../primitives" }
-pallet-ajuna-board        = { default-features = false, path = "../../pallets/ajuna-board" }
-pallet-ajuna-gameregistry = { default-features = false, path = "../../pallets/ajuna-gameregistry" }
-pallet-ajuna-matchmaker   = { default-features = false, path = "../../pallets/ajuna-matchmaker" }
-pallet-ajuna-runner       = { default-features = false, path = "../../pallets/ajuna-runner" }
+ajuna-common                 = { default-features = false, path = "../../ajuna-common" }
+ajuna-primitives             = { default-features = false, path = "../../primitives" }
+pallet-ajuna-awesome-avatars = { default-features = false, path = "../../pallets/ajuna-awesome-avatars" }
+pallet-ajuna-board           = { default-features = false, path = "../../pallets/ajuna-board" }
```

Comment:
Do we need `ajuna-board` ?

Reply:
sorry, it was already there, I just change the number of spaces before the = and it appears in the diff